skfda/exploratory/visualization/_ddplot.py

```diff
@@ -0,0 +1,142 @@
+"""DD-Plot Module.
+
+This module contains the necessary functions to construct the DD-Plot.
+To do this depth is calculated for the two chosen distributions, and then
+a scatter plot is created of this two variables.
+"""
+
+from typing import Callable
+from ... import FDataGrid
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
+from ._utils import (
+    _get_figure_and_axes,
+    _set_figure_layout_for_fdata,
+)
+
+
+class DDPlot:
+
+    """DDPlot visualization. Plot the depth of our fdata elements in two
+    different distributions, one in each axis. It is useful to understand
+    how our data is more related with one subset of data / distribution
+    than another one.
+
+    Args:
+        fdata: functional data set that we want to examine.
+        dist1: functional data set that represents the first distribution that
+            we want to use to compute the depth (Depth X).
+        dist2: functional data set that represents the second distribution that
+            we want to use to compute the depth (Depth Y).
+        depth_method: method that will be used to compute the depths of the
+            data with respect to the distributions.
+
+    """
+
+    def __init__(
+        self,
+        fdata: FDataGrid,
+        dist1: FDataGrid,
+        dist2: FDataGrid,
+        depth_method: Callable,
+    ) -> None:
+        self.fdata = fdata
+        self.dist1 = dist1
+        self.dist2 = dist2
+        self.depth_method = depth_method
+        self.depth_method.fit(fdata)
+
+    def plot(
+        self,
+        chart: Figure = None,
+        *,
+        fig: Figure = None,
+        axes: Axes = None,
+        n_rows: int = None,
+        n_cols: int = None,
+        width_aux_line: float = 0.35,
```

Comment:
I think that plotting parameters should not be specified in the `plot` method.

Reply:
Which parameters do you mean? Axes, n_rows, n_cols, width_aux_line...

Reply:
Colors and sizes.